test/OpenTelemetry.Tests/Impl/DistributedContext/Propagation/DistributedContextDeserializationTest.cs

```diff
@@ -284,7 +267,7 @@ private static void EncodeTagToOutPut(String key, String value, MemoryStream out
         private static void EncodeString(String input, MemoryStream output)
         {
             var length = input.Length;
-            var bytes = new byte[VarInt.VarIntSize(length)];
+            var bytes  = new byte[VarInt.VarIntSize(length)];
```

Comment:
extra whitespace 😄

Reply:
Yeah, I have local alignment settings set to make all adjacent assignments aligned. Will fix it.